qemu/tests/cfg/uefi_check_boot_option.cfg

```diff
@@ -0,0 +1,32 @@
+- uefi_check_boot_option:
+    only q35
+    only ovmf
+    type = uefi_check_boot_option
+    timeout = 360
+    wait_for_log_timeout = 60
+    start_vm = no
+    boot_menu = on
+    serial_login = no
+    last_error = "lasterror.*=.*0x0"
+    check_info_cmd = "grep PlatformBootManagerBeforeConsole %s"
+    ovmf_log = "ovmf.log"
+    extra_params += " -global isa-debugcon.iobase=0x402 -debugcon file:%s"
+    variants:
+        - time_10:
+            extra_params += " -boot menu=on,splash-time=10000"
```

Comment:
Let's add the option \"splash-time\" in avocodo-vt, I will send a patch.  And then set \"splash-time = 1000\"

Reply:
OK,thanks!